data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,325 @@
+			`	You quickly explain what happened. The Quarg seems very relieved about the fact that you were not able to talk to them.`
+			label explanation
+			`	"You ran into a most peculiar situation. This species should not be able to cross the borders of their system yet." It tilts it head for a moment considering what you just said. "It is our desire to help them, but it is too early for us to reveal ourselves to their species. I fear there is nothing we can do." It shakes its head.`
+			choice
+				`	"Then let me help them."`
+					goto initiative
+				`	"How can you make such a cold-hearted decision?"`
+			`	"This might be difficult for you to understand, but we must place the greater good above individual lives. If we show ourselves to them now, the shock of such an advanced species appearing in their space will fan the flames of chaos."`
+			choice
+				`	"Then let me help them."`
+					goto initiative
+				`	"You can deal with the consequences later. Help them now!"`
+			`	The Quarg takes on a sour look. "Insisting you are. Understand that our own intervention is out of the question." It sighs. "However, I can foresee the likelihood of you making a rash and impetuous decision to contact this ship."`
+			choice
+				`	"Sure, let me do the work."`
+				`	"If you are too weak..."`
+			`	Something like a laugh comes from the Quarg, before continuing.`
+			label initiative
+			`	"You lack the tools for such a mission. To my knowledge, no vessels capable of landing on gas giants have ever been developed by your kind. And you are certainly not able to make your intentions known to them in any meaningful sense."`
+			`	"But for the sake of the Incipias, I will assist you on this last point. If you cannot be dissuaded from your course of action, then it is best you are given a means of communication, so that your interactions can at least be on amicable terms. I can provide you with a single one of our translators, which will allow you to contact the species in their home system. Request that they sell you one of their ships, so that you are able to land on their planets."`
+			`	It pauses before taking on a more serious tone. "However, you must remember one thing. You are forbidden from granting them any advanced technology of yours. If you agree to that point we shall assist you."`
+			label choice
+			choice
+				`	"If that's how I can help them, sure."`
+					goto yes
+				`	"Yes, but why do you not want me to sell my tech?"`
+			`	"Halting the natural increase of a species is unpredictable at the best of times. Your species lacks the wisdom to understand this, as you are young, and have not seen what we have seen." The Quarg pauses for a moment, lost in thought. "The foreseeable risks of even letting you contact them are monumental, and not something you should consider trivial."`
```

Comment:
```suggestion
			`	"Speeding up the natural increase of a species is unpredictable at the best of times. Your species lacks the wisdom to understand this, as you are young, and have not seen what we have seen." The Quarg pauses for a moment, lost in thought. "The foreseeable risks of even letting you contact them are monumental, and not something you should consider trivial."`
```
The player would not be halting the increase if they granted technology.

Reply:
yes but they would halt the _natural_ increase

Reply:
Okay.